CHANGES.md

```diff
@@ -11,9 +11,12 @@
 - Added the `netbox_webhooks` sensor which listens for inbound webhooks from NetBox and fires triggers into StackStorm.
 
 ## 1.0.0
+
+- Drop Python 2.7 support
```

Comment:
Looking at the history, dropping python2 should go under the v2.0.0 changelog version.
The pack version was released correctly, it's just the CHANGES.md record is misleading.

See: 
https://github.com/StackStorm-Exchange/stackstorm-netbox/commit/4b8444574fe7c324db28a78c8c18eec5fed2e74d#diff-5472355e552efc17032084527cf9f00b21fda4841dc4996f2eca9fe7a5e3ada9